test/Microsoft.AspNet.Razor.Runtime.Test/TagHelpers/DefaultTagHelperContentTest.cs

```diff
@@ -301,7 +301,6 @@ public void Append_WithTagHelperContent_MultipleAppends()
 
             // Assert
             Assert.Equal(expected, copiedTagHelperContent.GetContent());
-            Assert.Equal(new[] { text1, text2 }, copiedTagHelperContent.ToArray());
```

Comment:
Why didn't you restore this i.e. `Assert.Equal(..., copiedTagHelperContent.Buffer.Entries)`?